<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMemoryZeile</class>
 <widget class="QWidget" name="WidgetMemoryZeile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>16</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="AdresseOut">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>45</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-right: 1px solid rgb(0,50,0);
border-bottom: 1px solid rgb(0,50,0);
background-color: rgb(100,100, 200);
color: rgb(170, 170, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">ADR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_0">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_1">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">01</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_2">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">02</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_3">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">03</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_4">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">04</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_5">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">05</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_6">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">06</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_7">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">07</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_8">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">08</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_9">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">09</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_10">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">0A</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_11">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">0B</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_12">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">0C</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_13">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">0D</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_14">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(200, 200, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">0E</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditValue_15">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>0</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid gray;
background-color: rgb(180,180, 255);
</string>
   </property>
   <property name="text">
    <string notr="true">0F</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_0" native="true">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_1" native="true">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_2" native="true">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_3" native="true">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_4" native="true">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_5" native="true">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_6" native="true">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_7" native="true">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_8" native="true">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_9" native="true">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_10" native="true">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_11" native="true">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_12" native="true">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_13" native="true">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_14" native="true">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="WidgetBitLeiste" name="BitLeiste_15" native="true">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>15</y>
     <width>20</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="AdresseOut_2">
   <property name="geometry">
    <rect>
     <x>365</x>
     <y>0</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-left: 1px solid rgb(0,50,0);
border-bottom: 1px solid rgb(0,50,0);
background-color: rgb(100,100, 200);
color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string notr="true">Lesen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="AdresseOut_3">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>0</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-left: 1px solid rgb(0,50,0);
border-bottom: 1px solid rgb(0,50,0);
background-color: rgb(100,100, 200);
color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string notr="true">Schreiben</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetBitLeiste</class>
   <extends>QWidget</extends>
   <header>widget_bit_leiste.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
